### PR TITLE
adding pop quiz 2

### PR DESCRIPTION
**Pull request by billythakidd04**
adding pop quiz 2 and answers

### CODE REVIEW

**Review by vricci518 — APPROVED** [review]
That was the answer I gave

**Review by rbaia28 — APPROVED** [review]
Looks good, my comment is in there (Question 1, line 50)

**Review by nburd90 — APPROVED** [review]
All good.

**Review by eiman-kased — APPROVED** [review]
#2 was the answer I gave.

File: Pop_Quiz.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+1. What command can I use to list all files (including hidden files) in my current directory? `ls -a` @eiman-kased
+
+1. What command can I use to find out what directory I am in? `pwd` @WaymonBrown84
+
+1. What is the difference between `'` and `"` in PHP? @tfrazzini13
+
+1. What are the PHP values that equate to `FALSE`? 0, null, false, empty string @nburd90
+
+1. What goes in a `<head></head>` tag? @margeschrec
```
Comment (margeschrec) [inline Pop_Quiz.md]: all the metadata for the page, such as links to the css sheet, google fonts, bootstrap etc.

**Review by tfrazzini13 — APPROVED** [review]

**Review by cwlainson — APPROVED** [review]

**Review by gc824 — APPROVED** [review]

File: Pop_Quiz.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+1. Why would I use an html class property vs an id? id should apply to a single element, where class applies to a larger subset of elements without being tag specific. ie error class @CharleswithLove
+
+1. Explain what `$_POST` is in PHP. It is an array @gc824
+
+1. What html property is used to populate the `$_POST` array in PHP? @williamrockowl
```
Comment (ghost) [inline Pop_Quiz.md]: the `name` property.

File: Pop_Quiz.md (hunk 1 above)
Comment (billythakidd04) [reply Pop_Quiz.md]: fixed

**Review by margeschrec — APPROVED** [review]

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by WaymonBrown84 — APPROVED** [review]
okie dokie

**Review by CharleswithLove — APPROVED** [review]
#62 was I right ?

File: Pop_Quiz.md (hunk 2 above)
Comment (billythakidd04) [reply Pop_Quiz.md]: fixed

**Review by ghost — APPROVED** [review]

**Pull request merged**